src/main/java/frc/robot/classes/Auton.java

```diff
@@ -120,22 +123,46 @@ private Command resetOdometry(String initialTrajectory) throws NullPointerExcept
         );
     }
 
-    private Command highScoreSequence() {
+    private Command highScoreSequenceCone() {
         return new SequentialCommandGroup(
+        new InstantCommand(intakeSubsystem::setConeMode),
         new ParallelDeadlineGroup(
-            new SequentialCommandGroup(
-            armSubsystem.goToState(ArmState.HIGHTRANSITION),
-            armSubsystem.goToState(ArmState.HIGHSCORE)
-            ), 
+            armSubsystem.goToState(ArmState.HIGHSCORE),
             new StartEndCommand(intakeSubsystem::pull, intakeSubsystem::stop, intakeSubsystem)
         ),
+        new WaitCommand(1).deadlineWith(new RepeatCommand(new InstantCommand(armSubsystem::updateMotors))),
         armSubsystem.goToState(ArmState.HIGHDROP)
         );
     }
 
+    private Command highTransitionSequenceCone() {
+        return new SequentialCommandGroup(
+        new InstantCommand(intakeSubsystem::setConeMode),
+        new ParallelDeadlineGroup(
+            armSubsystem.goToState(ArmState.HIGHTRANSITION),
+            new StartEndCommand(intakeSubsystem::pull, intakeSubsystem::stop, intakeSubsystem)
+        )
+        );
+    }
+
+    private Command highScoreSequenceCube() {
+        return new SequentialCommandGroup(
+        new InstantCommand(intakeSubsystem::setCubeMode),
+        new ParallelDeadlineGroup(
+            armSubsystem.goToState(ArmState.HIGHTRANSITION),
+            new StartEndCommand(intakeSubsystem::pull, intakeSubsystem::stop, intakeSubsystem)
+        ),
+        intakeSubsystem.pushTimed(1, false)
+        );
+    }
+
     private Command delay(double seconds){
         return new WaitCommand(seconds).beforeStarting(() -> commandLog.append("Wait " + seconds + " seconds"));
     }
+
+    private Command timeToMoveCommand(double speed, double timeout) {
```

Comment:
Change name to indicate that this moves along a single axis.

Reply:
changed to translateY